sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/student-scholastic-standings.aest.controller.ts

```diff
@@ -38,4 +41,18 @@ export class ScholasticStandingAESTController extends BaseController {
       scholasticStandingId,
     );
   }
+
+  /**
+   * Get Scholastic Standing summary details.
+   * @param studentId student id to retrieve the scholastic standing summary details.
+   * @returns Scholastic Standing Summary details.
+   */
+  @Get("summary/student/:studentId")
+  async getScholasticStandingSummary(
+    @Param("studentId", ParseIntPipe) studentId: number,
+  ): Promise<ScholasticStandingSummaryDetailsAPIOutDTO> {
+    return this.scholasticStandingControllerService.getScholasticStandingSummary(
```

Comment:
Usually, the API should consider returning a `@ApiNotFoundResponse` if the student id, in this case, is not found.